pebble.go

```diff
@@ -0,0 +1,459 @@
+//go:build pebbledb
+// +build pebbledb
+
+package db
+
+import (
+	"bytes"
+	"fmt"
+	"path/filepath"
+
+	"github.com/cockroachdb/pebble"
+)
+
+// ForceSync
+/*
+This is set at compile time. Could be 0 or 1, defaults is 1.
+It forces using Sync for NoSync functions (Set, Delete, Write)
+
+Notice if ForceSync=0: performance will be better. However, there is an issue when upgrading.
+And the workaround (if using ForceSync=0):
+At the upgrade-block, the sdk will panic without flushing data to disk or closing dbs properly.
```

Comment:
I didn't get it. Isn't the solution to use `SetSync` in the SDK? instead of forcing all ops to be `sync`

Reply:
i agree. However, this isnt an issue with goleveldb which is the default db. And other dbs are experiments atm.

Reply:
Still, I think the right way is to patch Cosmos SDK, not leave this hack even though pebbledb is experimental. Please remove

Reply:
Sir, it could be a big problem for node operators who run nodes for different chains and different sdk versions.
Also when syching archive nodes from genesis requires a lot of upgrades.
This hack is not the best way to fix. But it works.